src/overlays/actors/ovl_En_Yb/z_en_yb.c

```diff
@@ -171,14 +174,14 @@ void EnYb_ChangeAnim(PlayState* play, EnYb* this, s16 animIndex, u8 animMode, f3
         if (animIndex != this->animIndex || animMode != ANIMMODE_LOOP) {
```

Comment:
```suggestion
        if ((animIndex != this->animIndex) || (animMode != ANIMMODE_LOOP)) {
```
Once seen, you can't unsee